src/components/User/KycRequests/KycRequest.List.vue

```diff
@@ -206,18 +184,13 @@ export default {
     return {
       isLoading: false,
       list: [],
-      view: {
-        mode: VIEW_MODES_VERBOSE.index,
-        userId: null,
-        scrollPosition: 0,
-      },
+      userId: null,
       filters: {
         state: 'pending',
         requestor: '',
         type: '',
         pendingTasks: '',
       },
-      VIEW_MODES_VERBOSE,
       REQUEST_STATES,
```

Comment:
Useless

Reply:
what actually do you mean? I don't understand which part of code is useless